Tests/Functional/MappingToElasticaTest.php

```diff
@@ -21,11 +21,13 @@ class MappingToElasticaTest extends WebTestCase
     public function testResetIndexAddsMappings()
     {
         $client = $this->createClient(array('test_case' => 'Basic'));
-        $resetter = $this->getResetter($client);
-        $resetter->resetIndex('index');
 
         $type = $this->getType($client);
         $mapping = $type->getMapping();
+        $resetter = $this->getResetter($client);
```

Comment:
It seems that by juggling the location of the resetter, ```$this->getType($client)``` is using an index that hasn't been created yet. Resetting this to the original position fixes the current test error.